app/src/main/res/layout/activity_new_health_facility.xml

```diff
@@ -9,33 +9,35 @@
     <TextView
         android:id="@+id/text_health_facility_name_hint"
         style="@style/TextHint"
-        android:text="@string/text_health_facility_name_hint"
+        android:labelFor="@id/text_health_facility_name_field"
```

Comment:
I think that here the `android:text` should be kept as well, along with the new attribute.

Reply:
Yes, You are correct.